src/List.js

```diff
@@ -140,15 +141,15 @@ const Item = ({
 
   return (
     <ItemWrapper>
-      <TitleWrapper>
+      <TitleWrapper type='button' danger onClick={() => setOpen(!open)}>
```

Comment:
Adding `type='button'` will not be enough. You must add `as="button"` to convert the element to a button. See here: https://styled-components.com/docs/api#as-polymorphic-prop